src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -15,28 +15,29 @@ using namespace Windows::Foundation;
 using namespace Windows::System;
 using namespace Windows::ApplicationModel::DataTransfer;
 
-unsigned long long maxOperandNumber;
-
 String^ CopyPasteManager::supportedFormats[] =
 {
     StandardDataFormats::Text
 };
 
 constexpr wstring_view c_validCharacterSet{ L"0123456789()+-*/.abcdefABCDEF" };
+
+// The below values can not be "constexpr"-ed,
+// as both wstring_view and wchar[] can not be concatenated
 // [\s\x85] means white-space characters
 static const wstring c_wspc = L"[\\s\\x85]*";
-static const wstring c_wspcLParens = c_wspc + L"[(]*" + c_wspc;
-static const wstring c_wspcLParenSigned = c_wspc + L"([-+]?[(])*" + c_wspc;
-static const wstring c_wspcRParens = c_wspc + L"[)]*" + c_wspc;
-static const wstring c_signedDecFloat = L"[-+]?\\d*(\\d|[.])\\d*";
+static const auto c_wspcLParens = c_wspc + L"[(]*" + c_wspc;
+static const auto c_wspcLParenSigned = c_wspc + L"([-+]?[(])*" + c_wspc;
+static const auto c_wspcRParens = c_wspc + L"[)]*" + c_wspc;
+static const auto c_signedDecFloat = L"[-+]?\\d*(\\d|[.])\\d*";
```

Comment:
It's probably better to not use `auto` for this kind of variables, it can create some confusions and can introduce some issues.

A good example is these 4 lines, `c_wspcLParens`, `c_wspcLParenSigned`, `c_wspcRParens` are `wstring` while `c_signedDecFloat` is now a `const wchar_t[]`